src/procedures/procedure.c

```
@@ -154,6 +156,11 @@ bool Procedure_IsReadOnly(const ProcedureCtx *proc) {
 	return proc->readOnly;
 }
 
+const char *Procedure_GetDescription(const ProcedureCtx *proc) {
+    ASSERT(proc != NULL);
```

cppcheck

[error] Null pointer dereference

(nullPointer)